py/runai_model_streamer/runai_model_streamer/safetensors_streamer/safetensors_streamer.py

```diff
@@ -7,9 +7,19 @@
 
 RUNAI_DIRNAME = "RUNAI_DIRNAME"
 RUNAI_DIRNAME_TO_REMOVE = "RUNAI_DIRNAME_TO_REMOVE"
+GCS_PROTOCOL_PREFIX = "gs://"
+S3_PROTOCOL_PREFIX = "s3://"
+AWS_ENDPOINT_URL_ENV = "AWS_ENDPOINT_URL"
+DEFAULT_GCS_ENDPOINT_URL = "https://storage.googleapis.com"
 
 
 def convert_path_if_needed(path: str) -> str:
+    # Auto-configure settings if GCS prefix is specified.
```

Comment:
Does this still require Interperability setting on GCS or it is not required anymore?

Reply:
Yes, users would still need to specify HMAC keys via environment variables. Interoperability is configured here: https://cloud.google.com/storage/docs/authentication/hmackeys

Updated this comment to make this more clear.